addons/mass_mailing/report/mailing_trace_report_views.xml

```diff
@@ -62,6 +62,20 @@
            <field name="help" type="html"><p>Mass Mailing Statistics allows you to check different mailing related information like number of bounced mails, opened mails, replied mails. You can sort out your analysis by different groups to get accurate grained analysis.</p></field>
        </record>
 
+        <record id="mailing_trace_report_action_view_graph" model="ir.actions.act_window.view">
```

Comment:
Hello,

Why defining act_window view ? Is it only to specify mail views here and sms in _sms app ?

As those views are the only one available they will be used by default. In SMS you will have to give a different priority to SMS views (higher = not picked by default), and define an act_window_view for that specific one. It should lessen diff.

Reply:
Hello,

Indeed. The fact is if I don't also define an act_window in mass_mailing, it takes the pivot view redefined in mass_mailing_sms... weird, right ? I thought maybe it was because as a priority was defined in sms, it'll always take that view. So if I'm doing something wrong I'd be glad to have a little explanation about this :)

In summary :
- no act_window in mailing / no act_window in sms => mailing AND sms take mailing view
- no act_window in mailing / act_window in sms => mailing AND sms take sms view
- act_window in both => mailing takes mailing view, sms takes sms view